src/pytest_mock_resources/fixture/database/relational/generic.py

```diff
@@ -4,6 +4,7 @@
 import attr
 import six
 from sqlalchemy import MetaData
+from sqlalchemy.ext.asyncio import create_async_engine, AsyncSession
```

Comment:
These imports will also fail <1.4. Maybe it makes sense to route these through the `ImportAdaptor` compat mechanism